power_grid_model_io_native_c/power_grid_model_io_native/include/power_grid_model_io_native/pgm_vnf_converter/converter_parser.hpp

```diff
@@ -0,0 +1,98 @@
+// SPDX-FileCopyrightText: Contributors to the Power Grid Model project <powergridmodel@lfenergy.org>
+//
+// SPDX-License-Identifier: MPL-2.0
+
+#pragma once
+#ifndef POWER_GRID_MODEL_IO_NATIVE_C_CONVERTER_PARSER_HPP
+#define POWER_GRID_MODEL_IO_NATIVE_C_CONVERTER_PARSER_HPP
+
+#include <power_grid_model_io_native/common/common.hpp>
+
+#include <power_grid_model/auxiliary/input.hpp>
+#include <power_grid_model/container.hpp>
+
+#include <iostream>
+#include <map>
+#include <regex>
+
+namespace power_grid_model_io_native {
+using VisionGUID = std::string;
+
+struct VnfNode {
+    VisionGUID guid{"nan"};
+    double u_nom{nan};
+};
+
+using VnfGrid = pgm::Container<VnfNode>;
+template <typename Identifier> using IdentifierLookup = std::map<Identifier, ID>;
```

Comment:
`std::unordered_map` because we will be appending often and comparatively rarely use random element access (usually, we will just be iterating)